R/Data-validity.R

```diff
@@ -16,10 +16,10 @@ NULL
 #'   `ID` and `cohort` vectors which are of the same length `nObs`.
 validate_subjects <- function(object) {
   o <- Validate()
-  o$check(
-    test_int(object@nObs),
-    "nObs must be of type integer of length 1"
-  )
+  if (!test_int(object@nObs)) { # in if clause so that below test_* won't fail
+    o$check(FALSE, "nObs must be of type integer of length 1")
+    return(o$result())
```

Comment:
here I would directly put the simpler
```suggestion
    return("nObs must be of type integer of length 1")
```

Reply:
Sure, but that would assume that there will be no tests defined before this check.....